array/are.test.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-env jest */
+// @ts-ignore ambiguous import
+import are from "./are.ts";
+
+describe("are", () => {
+  it("returns true if the given arguments are all arrays", () => {
+    expect(are([2, 3])).toBe(true);
+    expect(are([1, 2, 3], [])).toBe(true);
+    expect(are([1, 2, 3], 8, [1, 3], "test")).toBe(false);
+  });
+
+  it("returns true for empty list of arguments", () => {
+    expect(are()).toBe(true);
+  });
```

Comment:
why it returns true for no arguments?

I would expect to return true only when arrays are passed. 
False for array mixed with other data types, no arguments, null, undefned, Number, String as argument.

Reply:
Just to be consistent with `[].every(Array.isArray)` which is what the `are` is meant to do. This is just a plural of `array/is` function.